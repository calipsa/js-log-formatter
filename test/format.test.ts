import formatter from '../src'

describe('format', () => {
  it('should output correct formatted log', () => {
    const result = formatter({
      name: 'some project',
      hostname: 'host',
      pid: 1111,
      v: 0,
      level: 40,
      time: new Date('2020-02-04T13:53:15.520Z'),
      msg: 'foobar',
      // @ts-ignore
      extra: 'qux',
    })
    expect(result).toMatchSnapshot()
  })
})
