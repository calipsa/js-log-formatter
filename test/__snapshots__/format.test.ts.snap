// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`format should output correct formatted log 1`] = `"[2m[13:53:15.520][22m [33m WARN[39m: foobar {\\"extra\\":\\"qux\\"} "`;
